smaug/core/types.proto

```diff
@@ -3,79 +3,80 @@ syntax = "proto3";
 package smaug;
 
 enum DataType {
-  UnknownDataType = 0;
```

Comment:
Can we restore the indent? If you really want to change this, please change all the proto files in another PR.

Reply:
+1

Reply:
Of course.